openapi.yaml

```diff
@@ -12463,6 +12465,45 @@ components:
                 a relative URL, it is relative to the domain you retrieved the
                 Event from.
               example: /v4/support/tickets/11111
+        secondary_entity:
+          type: object
+          readOnly: true
+          description: |
+            Detailed information about the Event's secondary entity, which provides additional information
+            for `linode_boot`, `linode_create`, and `linode_clone` Event actions.
+          x-linode-cli-display: 5
+          properties:
+            id:
+              type: integer
+              description: |
+                The unique ID for an Event's secondary entity.
+
+
+                Secondary Entities are assigned the ID of the Linode they correspond to.
+              example: 1234
+            label:
+              type: string
+              description: >
+                The current label of this object. The label may reflect changes
+                that occur with this Event.
+              example: linode1234
+            type:
+              type: string
+              enum:
+              - backup
+              - image
+              - linode
```

Comment:
This could be _any_ entity type.  Right now secondary entity specifically will only come back with "linode", "image", "linode_config", or "backup", but there may be more in the future.

Reply:
Cool - I'll make this a regular string - Also - noticed that this was "backup" in this test but other places in the code this was was "backups"...